appengine/standard/migration/memorystore/main.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This code contrasts how Python 2.7 apps using the memcache library can be
+# changed to use the Memorycache for Redis API and libraries, instead, under
+# either Python 2.7 or Python 3.6 or later.
+#
+# The examples are taken from the appengine/standard/memcache/snippets example
+# in this repository. The lines of code from the old example are included here
+# as comments, preceded by ## to distinguish them.
+#
+# The samples are wrapped in a Flask web app to run in App Engine.
+
+## from google.appengine.api import memcache
+import os
+import redis
+import time
+
+from flask import Flask, redirect, render_template, request
+
+redis_host = os.environ.get('REDISHOST', 'localhost')
```

Comment:
Nit: would an underscore (e.g. `REDIS_HOST`) be cleaner here?

Reply:
I'll change it.